examples/hybrid/types.jl

```diff
@@ -130,16 +132,39 @@ function get_spaces(parsed_args, params, comms_ctx)
     dz_bottom = FT(parsed_args["dz_bottom"])
     dz_top = FT(parsed_args["dz_top"])
     topography = parsed_args["topography"]
+
+    if parsed_args["topography"] != "NoWarp"
+        include("topography_helper.jl")
+    end
```

Comment:
I don't think files can / should be included inside functions. Can we unconditionally include and then conditionally call instead?

Reply:
Removed from function scope